### PR TITLE
[SEDONA-97] Add ST_Force_2D function to Apache Sedona

### PR DESCRIPTION
**Pull request by tmodi7**
## Is this PR related to a proposed Issue?
[SEDONA-97](https://issues.apache.org/jira/browse/SEDONA-97)
## What changes were proposed in this PR?
ST_Force_2D function
## How was this patch tested?
I have added unit tests in Java, Scala and python.
## Did this PR include necessary documentation updates?
Yes.

### CODE REVIEW

File: docs/api/sql/Function.md [hunk 1]
```diff
@@ -1093,8 +1093,8 @@ Example:
 
 ```SQL
 SELECT ST_AsText(
-    ST_Reverse(ST_GeomFromText('LINESTRING(0 0, 1 2, 2 4, 3 6)'))
-) AS geom
+               ST_Reverse(ST_GeomFromText('LINESTRING(0 0, 1 2, 2 4, 3 6)'))
```
Comment (jiayuasu) [inline docs/api/sql/Function.md:1096]: You changed all indents in the docs. This is NOT acceptable and will break the documentation website. Please only change the place you are supposed to change.

File: docs/api/sql/Function.md [hunk 2]
```diff
@@ -909,21 +909,21 @@ Result:
...
 `ST_Collect(*geom: geometry)`
 
 `ST_Collect(geom: array<geometry>)`
 
 Since: `v1.2.0`
 
-Example: 
+Example:
 
 ```SQL
```
Comment (jiayuasu) [inline docs/api/sql/Function.md:922]: Do NOT change other places that are NOT written by you.

File: docs/api/flink/Function.md [hunk 3]
```diff
@@ -19,18 +19,18 @@ Introduction:
 Transform the Spatial Reference System / Coordinate Reference System of A, from SourceCRS to TargetCRS
 
 !!!note
```
Comment (jiayuasu) [inline docs/api/flink/Function.md:21]: Do NOT change other places that are NOT written by you.

**Review by jiayuasu — CHANGES_REQUESTED** [review]

File: docs/api/sql/Function.md [hunk 4]
```diff
@@ -106,10 +106,10 @@ Introduction:
 Transform the Spatial Reference System / Coordinate Reference System of A, from SourceCRS to TargetCRS
 
 !!!note
-	By default, this function uses lat/lon order. You can use ==ST_FlipCoordinates== to swap X and Y.
+        By default, this function uses lat/lon order. You can use ==ST_FlipCoordinates== to swap X and Y.
```
Comment (jiayuasu) [inline docs/api/sql/Function.md:109]: You still changed the doc. This should be a tab. No change.

Comment (tmodi7) [reply docs/api/sql/Function.md]: Just did the right commit. Should be good now

File: docs/api/sql/Function.md [hunk 5]
```diff
@@ -1105,4 +1105,30 @@ Result:
...
+
+Format: `ST_Force_2D (A:geometry)`
+
+Since: `v1.2.1`
+
+Example:
+
+```SQL
+SELECT ST_AsText(
+    ST_Reverse(ST_GeomFromText('POLYGON((0 0 2,0 5 2,5 0 2,0 0 2),(1 1 2,3 1 2,1 3 2,1 1 2))'))
```
Comment (jiayuasu) [inline docs/api/sql/Function.md:1122]: Why do you use ST_Reverse here?

Comment (tmodi7) [reply docs/api/sql/Function.md]: Sorry Typo. Corrected it.

**Comment by jiayuasu** [comment]
Please update your PR description according to other PRs.

**Review by jiayuasu — APPROVED** [review]

**Pull request merged**